android/source/VideoLocker/src/org/edx/mobile/view/CourseChapterListFragment.java

```diff
@@ -443,7 +448,7 @@ private void hideProgressDialog(){
     private long lastClickTime;
     
     protected void showLastAccessedView(View v) {
-        if (v != null) {
+        if (v != null && isActivityStarted()) {
             if (!AppConstants.offline_flag) {
                 try {
                     if(courseId!=null && lastAccesed_subSectionId!=null){
```

Comment:
Nitpick: spelling error: `lastAccesed_subSectionId` should be `lastAccessed_subSectionId`.